### PR TITLE
cmd,client,daemon: add snap abort command

### PR DESCRIPTION
**Pull request by niemeyer**

### CODE REVIEW

File: daemon/api.go [hunk 1]
```diff
@@ -153,6 +153,7 @@ var (
 		Path:   "/v2/changes/{id}",
 		UserOK: true,
 		GET:    getChange,
+		POST:   abortChange,
```
Comment (zyga) [inline daemon/api.go]: We should update ``docs/rest.md`` later

Comment (niemeyer) [inline daemon/api.go]: Next week! :)

**Comment by pedronis** [comment]
LGTM

**Comment by zyga** [comment]
Looks good

**Pull request merged**